app/src/main/java/com/andrew/tasky/core/data/SharedPrefsImpl.kt

```diff
@@ -0,0 +1,41 @@
+package com.andrew.tasky.core.data
+
+import android.content.SharedPreferences
+import com.andrew.tasky.core.domain.SharedPrefs
+
+class SharedPrefsImpl(
+    private val sharedPrefs: SharedPreferences
+) : SharedPrefs {
+
+    override fun matchesUserId(s: String): Boolean {
```

Comment:
I'd avoid using such abbreviations. What is `s`? I assume it's just a user ID? Then I'd call it like that.